### PR TITLE
OHIF-310: Add report title dialog to set series description

### PR DESCRIPTION
**Pull request by igoroctaviano**
Screen Shot 2020-07-27 at 16 53 57

### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by igoroctaviano** [comment]
> Screen Shot 2020-07-27 at 16 53 57
> 
> ### PR Checklist
> * [ ]  Brief description of changes
> * [ ]  Links to any relevant issues
> * [ ]  Required status checks are passing
> * [ ]  User cases if changes impact the user's experience
> * [ ]  `@mention` a maintainer to request a review

**Comment by igoroctaviano** [comment]
@JamesAPetts I'll need you to hit the hammer if this is going to be a DCMJS change or we can intercept naturalized values passing them to the store function like done here.

**Comment by JamesAPetts** [comment]
`dcmjs` is modular enough that there is no real reason we can't do this, everything in `dcmjs` can be thought of like a helper tool, you can just write your own naturalized sets from scratch if you wanted.

You can currently pass options in to generate report but they don't do anything, however it would be nice to change this in the longer term. I think this is fine for now.

File: platform/core/src/DICOMSR/dataExchange.js [hunk 1]
```diff
@@ -86,12 +87,14 @@ const generateReport = (measurementData, additionalFindingTypes) => {
  * @param {object} dataSource The dataSource that you wish to use to persist the data.
  * @param {string[]} additionalFindingTypes toolTypes that should be stored with labels as Findings
  * as opposed to Finding Sites.
+ * @param {object} naturalizedReportValues Naturalized report tags previsouly defined.
```
Comment (swederik) [inline platform/core/src/DICOMSR/dataExchange.js:90]: typo

**Comment by JamesAPetts** [comment]
This is actually basically all implemented with Danny's state machine/workflow PR here: 

https://github.com/OHIF/Viewers/pull/1931

I cherry picked the `SeriesDescription` part of this PR and added it there.

**Pull request closed without merging**